web/tests/test_forms.py

```
@@ -564,3 +571,258 @@ def test_invalid_profile_form(self):
+        cls.mock_captcha = cls.captcha_patcher.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.captcha_patcher.stop()
+        super().tearDownClass()
+
+    def setUp(self):
+        self.User = get_user_model()
+        # Create a test user
+        self.user = self.User.objects.create_user(
+            username="testuser", email="testuser@example.com", password="testpass123"
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)